tests/integration/component_test/ts-consumer/Pulumi.yaml

```diff
@@ -0,0 +1,10 @@
+name: ts-consumer
+description: A minimal TypeScript Pulumi program
+runtime:
+  name: nodejs
```

Comment:
Consume in YAML instead for simplicity?

Reply:
I think it's preferable to have a program where we actually get output from `pulumi package add`, makes the test easier to write.

Reply:
I don't insist but curious - how does `package add` make the test easier to write compared to a declarative section?

Reply:
Actually, I framed my question wrong. The TS program looks empty - why wouldn't an empty YAML program be fine?

Reply:
We check against the output of `package add`, which tells us what we need to import.  That in turn contains the namespace and the name, making it easy to check against.